woocommerce/woocommerce
# Add @woocommerce/block-templates dependencies

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds the dependencies to `@woocommerce/block_templates` that are needed for the changes coming in #40722.

This was split out into its own to troubleshoot build issues I was facing in #40722.

This PR builds without issue because it leaves out the `@types/wordpress__data` devDependency, which causes conflicts and is not needed.

This PR should be reviewed and merged separately to keep the size of #40722 smaller and more focused on the actual code changes.



### How to test the changes in this Pull Request:



There are no changes to the actual codebase. Therefore, there should be no functional changes. This just updates the build to get ready for the changes in #40722.

1. Verify build succeeds.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
